src/turbomind/kernels/attention/rotary_embedding.h

```diff
@@ -67,6 +67,21 @@ __device__ void ApplyRotaryEmbedding(Array<T, 4>& x, float base, int dims, int t
     }
 }
 
+struct PrecomputeFastRoPE {
```

Comment:
Can we rewrite the `struct FastRoPE` instead of creating a new struct? cc @lzhangzz